MAINTAINERS.md

```diff
@@ -60,6 +62,7 @@ If you are an owner of the organization, you should be able to see an automated
 - Matthew Peveler ([@MasterOdin](https://github.com/MasterOdin)):
   [09 January 2021](https://github.com/tldr-pages/tldr/issues/5122) — [01 April 2021](https://github.com/tldr-pages/tldr/issues/5473)
 
+
```

Comment:
why the double enter?

Reply:
Yeah, let's clear that out @Waples:
```suggestion
```

Reply:
Everywhere else in the markdown document has double breaks between <subject>, why not here?

Reply:
Owh nevermind, I had a diff left over where I fixed all those double breaks. Want me to fix that or remove this ?

Reply:
Oh, send the fix in another PR :man_shrugging:

Reply:
I would've preferred that we fixed this here, because I see no reason for a newline being added where it was 🤔 

@Waples will you please open a PR to fix this?

Reply:
I trusted too much in this comment https://github.com/tldr-pages/tldr/pull/5775#discussion_r615058352 my fault.

Reply:
Don't worry, it's no problem.  I think @Waples made a mistake because there were actually _no_ double line-breaks in the file before this PR.